spec/change_set_persisters/hyrax/change_set_persister_spec.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literals: true
+
+RSpec.describe Hyrax::ChangeSetPersister, type: :model do
+  let(:adapter) { Valkyrie::MetadataAdapter.find(:indexing_persister) }
+  let(:persister) { adapter.persister }
+  let(:query_service) { adapter.query_service }
+  let(:storage_adapter) { Valkyrie.config.storage_adapter }
+  let(:change_set_persister) { Hyrax::ChangeSetPersister.new(metadata_adapter: adapter, storage_adapter: storage_adapter) }
```

Comment:
Would you mind using `described_class` here?

Reply:
Done